scripts/criu-ns

```diff
@@ -124,6 +126,7 @@ def wrap_restore():
     criu_pid = os.fork()
     if criu_pid == 0:
         os.setsid()
+        fcntl.ioctl(0, termios.TIOCSCTTY, 1)
```

Comment:
pls write a comment that explains what we are doing here and why do we need to do this

Reply:
Done, also I updated the patch logic a bit to skip this step when not needed (not shell job) and fail on notatty.